public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -66,7 +66,9 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
 
   // Important to memoize stuff here, as panel rerenders a lot for example when resizing.
   const [logRows, deduplicatedRows, commonLabels] = useMemo(() => {
-    const newResults = data ? dataFrameToLogsModel(data.series, data.request?.intervalMs) : null;
+    const newResults = data
```

Comment:
Just a nit and I know this is not introduced in this PR, but I am kinda unhappy with the name of this variable.